### ISSUE TITLE
Convert our ADRs to scope42 format

### ISSUE THREAD

**Issue by erikhofer** [scope42/scope42#224]
Currently, we have some Markdown ADRs. These sould be converted to scope42.

We could do this after #223 but it may take a while.

**Comment by erikhofer** [comment]
We have some two-dimensional ADRs, e.g. https://github.com/scope42/scope42/blob/main/docs/decisions/0003-item-identifiers.md

We may have to split this into two seperate ADRs 🤔 we should probably do this rather sooner than later.
